src/FormBuilder/resources/js/form-builder/src/containers/HeaderContainer.tsx

```diff
@@ -140,7 +149,12 @@ const HeaderContainer = ({
                     <TextControl
                         className={'givewp-form-title'}
                         value={formTitle}
-                        onChange={(formTitle) => dispatch(setFormSettings({formTitle}))}
+                        onChange={(formTitle) => {
+                            !isPublished &&
+                                formTitle !== 'Donation Form' &&
```

Comment:
@JoshuaHungDinh, Are we sure about this string comparison? Will it remain the same in different languages?

Reply:
Yea I think this can be done better. We don't need to match it in the onChange, if the formTitle is changing then it's redundant.

Because we only need to pass the formTitle to the pageSlugControl when an un-published form has an updated title. I've updated the conditional by checking if `cleanForSlug(formTitle)` matches `pageSlug`. The only time the form title could match the pageSlug is if the formTitle was updated.   Resolved: 4fb4931f139ee44506439d9fcc1d13fc480bbe5a.  (Missed the ```!``` on the isPublished check 3710522661217852fe051c92921d3662b7e8d153)